core/app/models/spree/app_configuration.rb

```diff
@@ -25,6 +25,7 @@ class AppConfiguration < Preferences::Configuration
     preference :admin_interface_logo, :string, default: 'admin/logo.png'
     preference :admin_path, :string, default: '/admin'
     preference :admin_products_per_page, :integer, default: Kaminari.config.default_per_page
+    preference :admin_variants_per_page, :integer, default: Kaminari.config.default_per_page
```

Comment:
@tomurb let's have this preference in https://github.com/spree/spree/blob/master/backend/app/models/spree/backend_configuration.rb as `variants_per_page`

Reply:
done